src/OpenDiffix.Core.Tests/Aggregator.Tests.fs

```diff
@@ -0,0 +1,90 @@
+module OpenDiffix.Core.AggregatorTests
+
+open Xunit
+open FsUnit.Xunit
+
+open CommonTypes
+
+let diffixCount = AggregateFunction(DiffixCount, { AggregateOptions.Default with Distinct = false })
+let distinctDiffixCount = AggregateFunction(DiffixCount, { AggregateOptions.Default with Distinct = true })
+let diffixLowCount = AggregateFunction(DiffixLowCount, AggregateOptions.Default)
+
+let randomNullableInteger (random: System.Random) =
+  let x = random.Next(100) - 10 // ~10% chance to get a NULL
+  if x < 1 then Null else Integer(int64 x)
```

Comment:
This seems a bit convoluted. Alternatively:
```FS
  if random.Next(10) = 0 then Null else Integer(int64 random.Next(100))
```

Reply:
Yeah, looks better!